### ISSUE TITLE
TODOs for Initial Release

### ISSUE THREAD

**Issue by Drullkus** [Drullkus/Shizzel#1]
- [ ] Chiseling Bee (Chisels blocks randomly in the world) - Custom Texture with Chisel Stinger
- [ ] ME Chisel Terminal (All variations are available depending on # of blocks inside the ME system)

**Comment by TheCricket** [comment]
oh god

**Comment by BBoldt** [comment]
Chiseled mashed potatoes?

**Comment by scroyall** [comment]
ME Chisel Terminal would be particularly useful if it had an option to automatically chisel inserted blocks back to their unchiseled original block, if possible.

**Comment by theonecomment** [comment]
Chiseled chisels!

**Comment by Winterunmute** [comment]
Will the chiseled ME terminal work similarily to a storage controller coupled with a storage bus, like if you have 64 cobble in it you can remote the cobble as any kind of chisel-block that that can become?

**Comment by Sirithil** [comment]
I pair logistics pipes with AE2 for my remote requesting. Bonus points if you can make it so LP can see all the chiseled variants in the terminal  as well so I can remote request them.

**Comment by SJ1133** [comment]
Chiselable vanilla armor if possible. That would be cool to see :D

**Comment by chylex** [comment]
Throwable 3D chisels.

**Comment by Drullkus** [comment]
@JRgames No, it will behave as a normal Terminal except it will have an extra slot to the left side. It will be an interactive Terminal-Chisel hybrid GUI.

**Comment by Rukachan** [comment]
Don't forget to add a license please.

**Comment by Drullkus** [comment]
Done, @Rukachan
